recipes/sbg-ecom/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir, replace_in_file
+
+import os
+
+required_conan_version = ">=2.1"
+
+
+class SbgEComConan(ConanFile):
+    name = "sbgecom"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/SBG-Systems/sbgECom"
+    description = "C library used to communicate with SBG Systems IMU, AHRS and INS"
+    topics = ("sbg", "imu", "ahrs", "ins")
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+    
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    languages = "C"
```

Comment:
```suggestion
    implements = ["auto_shared_fpic"]
    languages = "C"
```

missed this line initially to silence the hooks :)